haskell-language-server.cabal

```diff
@@ -213,191 +216,193 @@ flag cabalfmt
-  if flag(alternateNumberFormat)
-    build-depends: hls-alternate-number-format-plugin ^>= 1.3
-    cpp-options: -Dhls_alternateNumberFormat
+  if flag(alternatenumberformat)
+    build-depends: hls-alternate-number-format-plugin ^>=1.3
+    cpp-options:   -Dhls_alternateNumberFormat
 
 common qualifyImportedNames
-  if flag(qualifyImportedNames)
+  if flag(qualifyimportednames)
     build-depends: hls-qualify-imported-names-plugin ^>=1.0
-    cpp-options: -Dhls_qualifyImportedNames
+    cpp-options:   -Dhls_qualifyImportedNames
 
 common codeRange
-  if flag(codeRange)
-    build-depends: hls-code-range-plugin ^>= 1.1
-    cpp-options: -Dhls_codeRange
+  if flag(coderange)
+    build-depends: hls-code-range-plugin ^>=1.1
+    cpp-options:   -Dhls_codeRange
 
 common changeTypeSignature
-  if flag(changeTypeSignature)
-    build-depends: hls-change-type-signature-plugin ^>= 1.1
-    cpp-options: -Dhls_changeTypeSignature
+  if flag(changetypesignature)
+    build-depends: hls-change-type-signature-plugin ^>=1.1
+    cpp-options:   -Dhls_changeTypeSignature
 
 common gadt
   if flag(gadt)
-    build-depends: hls-gadt-plugin ^>= 1.0
-    cpp-options: -Dhls_gadt
+    build-depends: hls-gadt-plugin ^>=1.0
+    cpp-options:   -Dhls_gadt
 
 common explicitFixity
-  if flag(explicitFixity)
-    build-depends: hls-explicit-fixity-plugin ^>= 1.1
-    cpp-options: -DexplicitFixity
+  if flag(explicitfixity)
+    build-depends: hls-explicit-fixity-plugin ^>=1.1
+    cpp-options:   -DexplicitFixity
 
 common explicitFields
-  if flag(explicitFields)
-    build-depends: hls-explicit-record-fields-plugin ^>= 1.0
-    cpp-options: -DexplicitFields
+  if flag(explicitfields)
+    build-depends: hls-explicit-record-fields-plugin ^>=1.0
+    cpp-options:   -DexplicitFields
 
 -- formatters
 
 common floskell
-  if flag(floskell) 
-    build-depends: hls-floskell-plugin ^>= 1.0
-    cpp-options: -Dhls_floskell
+  if flag(floskell)
+    build-depends: hls-floskell-plugin ^>=1.0
+    cpp-options:   -Dhls_floskell
 
 common fourmolu
   if flag(fourmolu)
-    build-depends: hls-fourmolu-plugin ^>= 1.1
-    cpp-options: -Dhls_fourmolu
+    build-depends: hls-fourmolu-plugin ^>=1.1
+    cpp-options:   -Dhls_fourmolu
 
 common ormolu
-  if flag(ormolu) 
-    build-depends: hls-ormolu-plugin ^>= 1.0
-    cpp-options: -Dhls_ormolu
+  if flag(ormolu)
+    build-depends: hls-ormolu-plugin ^>=1.0
+    cpp-options:   -Dhls_ormolu
 
 common stylishHaskell
-  if flag(stylishHaskell) 
-    build-depends: hls-stylish-haskell-plugin ^>= 1.0
-    cpp-options: -Dhls_stylishHaskell
+  if flag(stylishhaskell)
+    build-depends: hls-stylish-haskell-plugin ^>=1.0
+    cpp-options:   -Dhls_stylishHaskell
 
 common refactor
   if flag(refactor)
-    build-depends: hls-refactor-plugin ^>= 1.1
-    cpp-options: -Dhls_refactor
+    build-depends: hls-refactor-plugin ^>=1.1
+    cpp-options:   -Dhls_refactor
 
 library
-  import:           common-deps
-                    -- configuration
```

Comment:
Right, so this is what I remembered and is IMO a deal-breaker for `cabal-fmt`: it doesn't preserve comments properly. `cabal-gild` seems _better_ (but not perfect), so that's a strong preference for me.